ratpack-core/src/main/java/ratpack/server/internal/Slf4jNoBindingDetector.java

```diff
@@ -36,7 +36,8 @@ private static boolean init() {
     } catch (ClassNotFoundException e) {
       System.err.println(
         "WARNING: No slf4j logging binding found for Ratpack, there will be no logging output." + System.lineSeparator()
-          + "WARNING: Please add an slf4j binding, such as slf4j-log4j2, to the classpath."
+          + "WARNING: Please add an slf4j binding, such as slf4j-log4j2, to the classpath." + System.lineSeparator()
+          + "WARNING: More info may be found here: http://ratpack.io/manual/current/logging.html"
```

Comment:
@alkemist would it be better to mention `slf4j-simple` here instead of `slf4j-log4j2`? If a user chooses `slf4j-log4j2` then they will need to also include `log4j2` and configure it but `slf4j-simple` doesn't need any more dependencies and has default config that logs everything from `INFO` up and outputs it to `System.err`.